MetadataExtractor/Formats/Jpeg/JpegSegmentType.cs

```diff
@@ -221,7 +221,11 @@ public static bool CanContainMetadata(this JpegSegmentType type)
         }
 
         /// <summary>Gets JPEG segment types that might contain metadata.</summary>
+#if NET5_0_OR_GREATER
+        public static IReadOnlyList<JpegSegmentType> CanContainMetadataTypes { get; } = Enum.GetValues<JpegSegmentType>().Cast<JpegSegmentType>().Where(type => type.CanContainMetadata()).ToList();
```

Comment:
Is the ```Cast<JpegSegmentType>()``` needed on this path? (I think the GetValues<JpegSegmentType>() returns a JpegSegmentType[] already ?)